buildlib/tools/test_mad.sh

```diff
@@ -45,7 +46,11 @@ docker_run_srv() {
         -v /hpc/local:/hpc/local \
         --ulimit memlock=-1:-1 --device=/dev/infiniband/ \
         $IMAGE \
-        bash -c "${PWD}/install/bin/ucx_perftest -K ${HCA}"
+        bash -c "${PWD}/install/bin/ucx_perftest -K ${HCA} > ${PWD}/ucx_perf_srv.log 2>&1"
```

Comment:
since we are starting the server 2 times, we should probably generate 2 different logs for each test.

Reply:
Added unique names for the server log files to be on the safe side.